src/main/java/seedu/duke/data/Budget.java

```diff
@@ -4,23 +4,23 @@
 import static seedu.duke.common.Constants.MAX_AMOUNT_VALUE;
 import static seedu.duke.common.Constants.MIN_BUDGET_VALUE;
 
+//@@author wcwy
 /**
  * Represents the user's budget for the current month.
  *
  * <p>The default budget value is $1000, or the value read from the file storage.
  * Operations related to the budgets are also defined under this class.
  */
 public class Budget {
-    //@@author wcwy
     // Default value of the monthly budget is $1000
-    private static int budget = 1000;
+    private static long budget = 1000;
```

Comment:
This one can be changed to a defined constant, as 1000 for budget is a magic number.